### PR TITLE
feat: add devops locale

### PR DESCRIPTION
**Pull request by yazhouio**
### What type of PR is this?

/kind bug

### What this PR does / why we need it:
feat: add devops locale
### Which issue(s) this PR fixes:

Fixes #https://github.com/kubesphere/issues/issues/1198
### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note
None
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by yazhouio** [comment]
need pick to kse 3.4.1

File: locales/en/l10n-devopsProjects-pipeline-details.js [hunk 1]
```diff
@@ -231,6 +231,7 @@ module.exports = {
     // detail page -> pipeline configuration tab
   PIPELINE_CONFIGURATION: 'Pipeline Configuration',
```
Comment (Bettygogo2021) [inline locales/en/l10n-devopsProjects-pipeline-details.js:232]: Pipeline Configurations

File: locales/en/l10n-devopsProjects-pipeline-details.js [hunk 2]
```diff
@@ -231,6 +231,7 @@ module.exports = {
...
   Replay: 'Replay',
+  BRANCH_DISABLED_NOT_REPLAY: 'The branch is disabled and cannot be replayed.',
```
Comment (Bettygogo2021) [inline locales/en/l10n-devopsProjects-pipeline-details.js:234]: The branch has been disabled and cannot be replayed.

**Review by Bettygogo2021 — CHANGES_REQUESTED** [review]
pls change the trans.

**Comment by harrisonliu5** [comment]
@yazhouio need to update

**Pull request merged**